Slider/Assets/Scripts/UI/UIArtifactMenus.cs

```diff
@@ -38,6 +44,52 @@ private void Awake()
         Controls.RegisterBindingBehavior(this, Controls.Bindings.Player.CycleEquip, context => _instance.screenAnimator.CycleScreen()); // CONTROLER BIND ONLY
         //For controller support
         Controls.RegisterBindingBehavior(this, Controls.Bindings.UI.Back, context => _instance.CloseArtifact());
+
+        Player.OnControlSchemeChanged += ToggleNavigation;
+    }
+
+    private void ToggleNavigation(string s)
+    {
+        if (s == Controls.CONTROL_SCHEME_CONTROLLER)
+        {
+            EnableNavigation();
+        } 
+        else
+        {
+            DisableNavigation();
+        }
+    }
+
+    private void SaveSelectibles()
+    {
+        navMode = new();
+        selectibles = GetComponentsInChildren<Button>(true);
+        foreach(Button s in selectibles)
+        {
+            navMode.Add(s, s.navigation.mode);
+        }
+        ToggleNavigation(Controls.CurrentControlScheme);
+    }
+
+
+    private void DisableNavigation()
+    {
+        foreach(Selectable s in selectibles)
+        {
+            var nav = s.navigation;
+            nav.mode = Navigation.Mode.None;
+            s.navigation = nav;
```

Comment:
Does this need to be 3 lines

Reply:
it didn't like when i did read/write in one line 🤷